cms/admin/forms.py

```diff
@@ -820,7 +820,7 @@ def __init__(self, *args, **kwargs):
     def get_root_nodes(self):
         # TODO: this needs to avoid using the pages accessor directly
         nodes = TreeNode.get_root_nodes()
-        return nodes.exclude(cms_pages__is_page_type=not(self.page.is_page_type))
+        return nodes.exclude(cms_pages__is_page_type=not self.page.is_page_type)
```

Comment:
To prevent confusion, can we assign `not self.page.is_page_type` to a variable and use that in the query?

Reply:
@vinitkumar Yeah I suppose so, I was just trying to keep changes to a minimum. But that makes sense.